mix/qml/main.qml

```diff
@@ -279,12 +302,20 @@ ApplicationWindow {
 	Action {
 		id: saveAllFilesAction
 		text: qsTr("Save All")
-		shortcut: "Ctrl+S"
+		shortcut: "Ctrl+A"
```

Comment:
Ctrl+A is usually used to select all text. I Suggest using Ctrl+Shift+S instead.